index.rst

```diff
@@ -195,6 +195,27 @@ Summary
 Whilst the job-level pooling and externalized approaches will help with registry contention, by far the safest approach, and in the long term the simplest, is to adopt the limited read-only registry.
 This removes any worries about merging the outputs of multiple jobs and lets us leverage the knowledge already known to the graph builder.
 
+Task Breakdown
+--------------
+
+The initial prototype implementation will not include URL signing since all Data Preview 0.2 workflow executions will be run by staff.
+
+An initial break down of work would then be:
+
+* Add facility to pipetask command line to create a minimalist SQLite registry from the constructed quantum graph.
```

Comment:
Just to add a bit of detail that I think is important for how we approach this:
 - I think this belongs in `pipe_base` now, since that's where `QuantumGraph` lives.
 - The fundamental operations here are "export+import the dimension data and overall-input datasets" and "insert the intermediate and overall-output datasets into the new Registry".  I think I'd advocate just having this operate on two `Registry` instances, rather than go through the export/import YAML stuff for the first step.

Long term, I think it's a _really_ intriguing possibility that we could integrate this with QuantumGraph generation itself, and essentially populate the "scratch" SQLite database with the output datasets and the predicted provenance as we build the graph.  That could simplify a lot of code in both `daf_butler` and `pipe_base` for two reasons:
 - It could let us remove the idea of an unresolved `DatasetRef` entirely (`Quantum` is the only place we actually need them).  And that could make it easier to fully integrate `FakeDatasetRef` and `DatasetRef`, essentially making the `dataset_id` mandatory and the data ID and run optional (in some contexts).  And that could help with JSON serialization and client/server.
 - By writing predicted outputs as we predict them, I bet we can simplify QG generation logic both in `pipe_base` and in what it demands from `daf_butler`.

I don't want to push us going to that longer-term vision now - it's pretty disruptive, as it involves making developer-mode `pipetask` write to that local SQLite database, too.  But I'd like everyone to keep it in mind, at least as a reason not to overengineer or otherwise spend too much time one the initial implementation of this.